torchgeo/samplers/batch.py

```diff
@@ -88,12 +88,17 @@ def __init__(
             size: dimensions of each :term:`patch`
             batch_size: number of samples per batch
             length: number of samples per epoch
+                (defaults to the maximal number of non-overlapping :term:`chips <chip>`
+                of size ``size`` that can be sampled from the dataset)
```

Comment:
My only problems with this are:

1. It's technically not true, the last row/col will overlap
2. It may imply that the chips returned will not overlap, which is not the case

Reply:
1. Or something like, "the number of chips that would be sampled following the "tile_to_chips" strategy from ...."
2. If you want, you can also add a note that randomly sampled chips might overlap

Reply:
1. I feel like this is going to end up being a whole paragraph lol, that's why I originally said "reasonable default" and left it at that. Maybe just throw in an "approximately" somewhere